src/sphinx/formats/docker.rst

```diff
@@ -128,11 +128,16 @@ Environment Settings
   ``dockerChmodType``
     The file permissions for the files copied into Docker image when ``MultiStage`` or ``Run`` strategy is used.
 
-    * ``DockerChmodType.UserGroupReadExecute`` (default): chmod -R u=rX,g=rX
-    * ``DockerChmodType.UserGroupWriteExecute``: chmod -R u=rwX,g=rwX
-    * ``DockerChmodType.SyncGroupToUser``: chmod -R g=u
+    * ``DockerChmodType.UserGroupReadExecute`` (default): chmod u=rX,g=rX
+    * ``DockerChmodType.UserGroupRead``: chmod u=r,g=r
+    * ``DockerChmodType.UserGroupWriteExecute``: chmod u=rwX,g=rwX
+    * ``DockerChmodType.SyncGroupToUser``: chmod g=u
+    * ``DockerChmodType.UserGroupPlusExecute``: chmod u+x,g+x (This is for ``dockerAdditionalPermissions``)
     * ``DockerChmodType.Custom``: Custom argument provided by the user.
 
+  ``dockerAdditionalPermissions``
+    Additional permissions typically used to give ``chmod +x`` rights for the executable files. By default generated Bash scripts are given ``DockerChmodType``.
```

Comment:
Missing permissions?

Reply:
Added this via the UI :+1: thanks for reviewing this as well :hug